<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowOnAlarmLayoutActionWidget</class>
 <widget class="QWidget" name="ShowOnAlarmLayoutActionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>115</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QPushButton" name="selectUsersButton">
     <property name="text">
      <string notr="true">&lt;All users&gt;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QCheckBox" name="forceOpenCheckBox">
     <property name="text">
      <string>Force Alarm Layout opening</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="usersLabel">
     <property name="text">
      <string>for</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="nx::vms::client::desktop::RewindForWidget" name="rewindForWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>nx::vms::client::desktop::RewindForWidget</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/event_rules/widgets/rewind_for_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>selectUsersButton</tabstop>
  <tabstop>forceOpenCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
